src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,93 @@
 {
   "type": "CategoricalFusion",
-  "structural_elements": [
+  "structure": [
     {
       "type": "TranscriptSegmentElement",
       "transcript": "refseq:NM_004327.3",
-      "gene_descriptor": {
-        "type": "GeneDescriptor",
-        "id": "normalize.gene:BCR",
-        "gene_id": "hgnc:1014",
+      "gene": {
+        "type": "Gene",
+        "id": "hgnc:1014",
         "label": "BCR"
       },
-      "element_genomic_end": {
-        "id": "fusor.location_descriptor:NC_000022.11",
-        "type": "LocationDescriptor",
-        "label": "NC_000022.11",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000022.11",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 23253980
-            },
-            "end": {
-              "type": "Number",
-              "value": 23253981
-            }
-          }
-        }
+      "elementGenomicEnd": {
+        "id": "ga4gh:SL.5f5OVC3zReewA4S78X1eO0oJwWNbGYsY",
+        "type": "SequenceLocation",
+        "description": null,
+        "xrefs": null,
+        "alternativeLabels": null,
+        "extensions": null,
+        "sequenceReference": {
+          "id": "refseq:NC_000022.11",
+          "refgetAccession": "SQ.7B7SHsmchAR0dFcDCuSFjJAo7tX87krQ",
+          "type": "SequenceReference"
+        },
+        "start": 23253980,
+        "end": 23253981
```

Comment:
I think this should just be `"end": 23253980`

Reply:
I agree–did this get addressed?